controllers/configuration/backend/kubernetes.go

```diff
@@ -81,14 +81,14 @@ func newK8SBackendFromInline(backendConfig *ParsedBackendConfig, client client.C
 		ns = getDefaultK8sBackendSecretNS()
 	}
 	return &K8SBackend{
-		Client:       client,
+		Client:       ctx.k8sClient,
 		HCLCode:      renderK8SBackendHCL(suffix, ns),
 		SecretSuffix: suffix,
 		SecretNS:     ns,
```

Comment:
Is it necessary to wrap k8sClient in ctx? The function body didn't use any other fields of ctx.